package.json

```diff
@@ -53,10 +59,21 @@
       "json"
     ],
     "testMatch": [
-      "**/test.{ts,tsx}"
+      "**/src/**/test.{ts,tsx}",
+      "**/tools/*.test.ts"
     ],
     "transform": {
-      "\\.tsx?$": "<rootDir>/build/transform.js"
+      "/src/.+\\.tsx?$": "<rootDir>/build/transform.js",
+      "/tools/.+\\.ts$": "ts-jest/preprocessor"
+    },
+    "mapCoverage": true,
+    "coverageThreshold": {
+      "global": {
+        "branches": 100,
```

Comment:
I wouldn't force 100% coverage. I see you wanna provide the best quality, but sometimes is hard to achieve. What about 95% to have some flexibility?

Reply:
Done.

I usually set 100% but put some `// istanbul ignore next` on those no-need-to-test expressions. 😅

Reply:
That's a good thing as well! At the end, the coverage is kind of "feeling metric", so it makes a good sensation in users :)